packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -491,18 +468,117 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
                           ?? const SizedBox.shrink();
     return OverlayPortal.targetsRootOverlay(
       controller: _optionsViewController,
-      overlayChildBuilder: _buildOptionsView,
+      overlayChildBuilder: (BuildContext context) {
+        return _OptionsView<T>(
```

Comment:
Use a `ValueListenableBuilder` instead?

Reply:
also consider adding an assert to verify the constraints is not unbounded.

Reply:
Thanks, I forgot ValueListenableBuilder was a thing. Much cleaner.